prep-html-syntax/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Prep-HTML-Syntax</title>
+<h1 style="color: green;">My favorite character from Star Wars is Boba Fett!</h1>
```

Comment:
Hi Steve, the visible content of the page should go into the `<body></body>` of the document. So your `<h1></h1>` should get moved there.